src/main/scala/nl/knaw/dans/easy/stage/Conf.scala

```diff
@@ -7,14 +7,15 @@ import org.rogach.scallop.{singleArgConverter, ValueConverter, ScallopConf}
 
 class Conf(args: Seq[String]) extends ScallopConf(args) {
   printedName = "easy-stage-dataset"
-  version(s"$printedName ${Version()}")
+  val indent_____ = printedName.replaceAll(".", " ")
```

Comment:
Why the strange variable name with the underscores?

Optionally you could also replace the `printedName.replaceAll(".", " ")` with: `printedName.map(_ => ' ')`. 
I had to think twice about what `replaceAll(".", " ")` did (forgot that the first parameter was a pattern).